ideascube/library/migrations/0008_library_stock.py

```diff
@@ -163,7 +163,7 @@ class Migration(migrations.Migration):
             },
             bases=('monitoring.specimen', models.Model),
         ),
-        migrations.RunPython(migrate_data),
+        migrations.RunPython(migrate_data, hints={'using': 'default'}),
```

Comment:
Wasn't there a commit which made all the `RunPython` migrations reversible at some point in this branch?